uploadBundle.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+BUNDLE="$1"
+ENVIRONMENT="$2"
+
+require jq
+
+case $# in
+  2)
+    ENVIRONMENT="$2"
+    ;;
+  1)
+    ENVIRONMENT="local"
+    ;;
+  *)
+    echo "not enough arguments supplied.  You must supply the bundle directory to this command."
+    exit 1
+    ;;
+esac    
+
+source setEnvForUpload.sh $ENVIRONMENT
+
+safeName () {
```

Comment:
Would there need to be an equivalent for handling special characters in the names in downloadBundle.sh?

Reply:
no, because they're handled serverside.  The same logic is implemented in both places.